### PR TITLE
Hoc2015 play sound dropdown (and other fixes)

### PR DESCRIPTION
**Pull request by cpirich**
- fixed C-3PO spelling in blockly sound list
- turned off "show code" for starwarsblocks levels 7-15
- enhanced `playSoundDropdown()` function used to create the droplet `playSound` block's list to include a `click` handler that will play the sound when it is chosen from the dropdown list. (Note: this doesn't impact autocomplete in text mode or the blockly `playSound` block's dropdown)

### CODE REVIEW

**Comment by cpirich** [comment]
@breville @islemaster

**Comment by islemaster** [comment]
LGTM!

File: apps/src/studio/paramLists.js [hunk 1]
```diff
@@ -22,8 +31,29 @@ exports.getPlaySoundValues = function (withRandom) {
...
+        return metadata.name.toLowerCase().trim() === lowercaseSound;
+      }));
+
+      studioApp.playAudio(lowercaseSound, playbackOptions);
+      callback(quote(sound));
+    };
+    return {
+      text: quote(sound),
+      display: quote(sound),
+      click: handleChooseClick
```
Comment (breville) [inline apps/src/studio/paramLists.js:56]: neat.  droplet already had click callback support?

Comment (cpirich) [inline apps/src/studio/paramLists.js:56]: yes, fortunately, it did! Applab uses it for one menu item in the assert dropdown where you can choose to add a new image.

**Pull request merged**